src/runtime_src/core/pcie/driver/linux/xocl/subdev/clock.c

```diff
@@ -460,16 +856,9 @@ static int clock_ocl_freqscaling_impl(struct clock *clock, bool force,
 			0x00000002);
 
 		CLOCK_INFO(clock, "clockwiz waiting for locked signal");
-		mdelay(100);
-		for (j = 0; j < 100; j++) {
-			val = reg_rd(clock->clock_bases[i] +
-				OCL_CLKWIZ_STATUS_OFFSET);
-			if (val != 1) {
-				mdelay(100);
-				continue;
-			}
-		}
-		if (val != 1) {
+
+		err = clock_wiz_busy(clock, i, 100, 100);
```

Comment:
nit, will the log message looks like
"
clockwiz(%d) is (%u) busy after %d ms
clockwiz MMCM/PLL did not lock after %d ms, restoring the original configuration
"
Are the two "ms", %d and MMCM/PLL same thing? If so, can we skip something from the second log line?

Reply:
yeah. fixed.